src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.skia.cs

```diff
@@ -169,7 +175,7 @@ SkiaGeometrySource2D BuildGeometry()
 			spriteShape.Geometry = compositor.CreatePathGeometry(new CompositionPath(BuildGeometry()));
 
 			shapeVisual.Size = new Vector2((float)area.Width, (float)area.Height);
-			shapeVisual.Offset = new Vector3(0, 0, 0);
+			// shapeVisual.Offset = new Vector3((float)adjustedArea.Top, (float)adjustedArea.Left, 0);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)